index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta http-equiv="refresh" content="10">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Web3 Confernce website for capstone1</title>
+    <link rel="stylesheet" href="./Styles.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel=“stylesheet” href=“https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css”>
+    <script src="./Js/Speakers.js" defer></script>
+    <script src="./Js/Dynamics.js" defer></script>
+    <script src=”https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js” defer></script>
+<body>
+    <div class="top-bar">
+      <ul>
+        <li><a href="https://github.com/CrystallineButterfly"><img src="./Images/github.svg" alt="github"></a></li>
+        <li><a href="https://twitter.com/CrystAlline_K42"><img src="./Images/twitter.svg" alt="twitter"></a></li>
+        <li>code</li>
+        <li>login</li>
+        <li>language</li>
+      </ul>
+    </div>
+    <header>
+      <div class="container main-nav">
+        <div class="logo">Ω</div>
+        <nav class="all-pages">
+          <a href="/about.html" class="nav-link">About</a>
+          <a href="#" class="nav-link">Game</a>
+          <a href="#" class="nav-link">More Alpha</a>
+          <a href="#" class="nav-link">Hire us?</a>
+          <a href="#" class="nav-link">Sponsors</a>
+          <a href="#speakers" class="nav-link border">Speakers</a>
+        </nav>
+      </div>
+        <!-- MOBILE RESPONSIVE-->
+      <nav class="mobile-menu">
+        <!--Mobile Nav-->
+        <!-- <div class="mylogo">logo</div> -->
+        <div class="mobile-nav-bar">
+          <div class="btn-nav-open">
+            <button class="btn-open" type="button">
+              <img
+                class="btn-open-img"
+                src="./Images/Union.svg"
+                alt="hamburger menu"/>
+            </button>
+          </div>
+          <div class="mobile-nav-container">
+            <div class="btn-nav-close">
+              <button class="btn-close" type="button">X</button>
+            </div>
+            <div class="menu-list-items">
+              <a class="menu-title" href="./about.html">About</a>
```

Comment:
- [ ] I will suggest you to change the background color of your mobile navigation once we click on the hamburger button.
![image](https://user-images.githubusercontent.com/71874425/169612032-31b9a38b-d9f8-457f-adf8-5b9030a8a7aa.png)
This blue color doesn't match anything in your font colors. Kindly, change it to another color which match with your design.

Reply:
Hi @RolandM99 Cleaned up, thank you!